context-free.md

```diff
@@ -2,21 +2,61 @@
 
 ##  Who is this programming language for?
 
+Context Free is for artists iterested in exploring generative art.
 
 ## What is easy to do in this language? Why is it easy?
 
+It's easy to generate art that has a lot of symmetry. One of the easiest tricks
```

Comment:
I really agree with what Jeb mentioned about the ease in generating art with a lot of symmetry and recursive patterns. I can see it in his artwork!